script.js

```diff
@@ -24,9 +26,12 @@ let createElement = (newTodoObj) => {
   todoDiv.appendChild(todoLi);
   getTodoList().appendChild(todoDiv);
 }
+const modelTodos = [];
+
+}
 
+window.onload = dom;
```

Comment:
`dom` exists inside `Main`

you need to instantiate a `Main` instance with `new`